02-peer-protocol.md

```diff
@@ -348,6 +348,8 @@ The sender:
 avoid double-spending of the funding transaction.
   - MUST set `channel_reserve_satoshis` greater than or equal to `dust_limit_satoshis` from the `open_channel` message.
   - MUST set `dust_limit_satoshis` less than or equal to `channel_reserve_satoshis` from the `open_channel` message.
+  - if `option_channel_type` was negotiated:
+    - MUST set `channel_type`
   - if it sets `channel_type`:
```

Comment:
is this not the same as the requirement the two lines below? is `option_channel_type` being negotiated different from 'setting `channel_type`'?

```
  - if it sets `channel_type`:
```

is the same as 

```
- if `option_channel_type` was negotiated
```

Reply:
anyway, clightnng enforces the draft, which is "if the opener sends a channel_type, we reply with a channel type.

Reply:
I think it's not quite the same: we never explicitly say here that having negotiated `option_channel_type` means we must set it. But if I understand you correctly, instead of adding two new lines I could just enrich the two lines below to more explicitly say that you MUST set channel type to what was in the `open_channel`, is that right?

Reply:
Let me know if https://github.com/lightning/bolts/pull/960/commits/584b03ea0f962b3d9321f78334f30a5316727d40 is better, I modified the existing requirement to clarify it instead of adding a new one on top

Reply:
yep that's great -- thanks!